diffprivlib/mechanisms/snapping.py

```diff
@@ -4,8 +4,11 @@
 """
 import struct
 
-import crlibm
 import numpy as np
+try:
+    from crlibm import log_rn
+except ModuleNotFoundError:
+    log_rn = np.log
```

Comment:
A info level log message would be useful

Reply:
Thanks, I'll keep this comment on the todo list for the future.